isisd/isis_nb_config.c

```diff
@@ -1508,6 +1522,96 @@ int isis_instance_segment_routing_srgb_upper_bound_modify(
 	return NB_OK;
 }
 
+/*
+ * XPath: /frr-isisd:isis/instance/segment-routing/srlb
+ */
+int isis_instance_segment_routing_srlb_pre_validate(
+	struct nb_cb_pre_validate_args *args)
+{
+	uint32_t srgb_lbound;
+	uint32_t srgb_ubound;
+	uint32_t srlb_lbound;
+	uint32_t srlb_ubound;
+
+	srgb_lbound = yang_dnode_get_uint32(args->dnode, "../srgb/lower-bound");
+	srgb_ubound = yang_dnode_get_uint32(args->dnode, "../srgb/upper-bound");
+	srlb_lbound = yang_dnode_get_uint32(args->dnode, "./lower-bound");
+	srlb_ubound = yang_dnode_get_uint32(args->dnode, "./upper-bound");
+
+	/* Validate SRLB against SRGB */
```

Comment:
Done. I add the verification in the pre-validate functions. But, can you remember me where this limitation comes from? There is nothing about such limitation in the RFC8667 and don't see any particular limitation in the label manager. But, in any case, it is safe to avoid a CLI configuration request all the MPLS range.

Reply:
Indeed. It looks like 65535 is an artificial limit since the RFC specifies that the SRGB/SRLB range fields have three bytes. In any case it's good to have the CLI limits and NB validations consistent with each other.

Reply:
I understand that CLI is just a pass through of yang, so validation is already done by `isis_instance_segment_routing_srlb_pre_validate()` function since `segment-routing local-block` CLI command call yang functions. Am I wrong ?

Reply:
Yes you're right. My comment was that the label ranges accepted by the CLI should be in sync with the label ranges accepted by the NB, which is already the case. Marking as resolved... (forgot to do it before)